imports/plugins/included/marketplace/client/templates/stripeConnectSignupButton/stripeConnectSignupButton.js

```diff
@@ -29,6 +29,10 @@ Template.stripeConnectSignupButton.events({
 
     const shop = Shops.findOne({ _id: shopId });
 
+    if (!shop || !shop.workflow || shop.workflow.status !== "active") {
```

Comment:
I think this is probably a good intermediate fix. Long run we should be building up the shops workflow and aligning UI with that so that buttons that are inactive are not even available on the UI.